metricbeat/module/golang/expvar/_meta/docs.asciidoc

```diff
@@ -0,0 +1,6 @@
+=== golang expvar MetricSet
+
+This is the expvar metricset of the module golang.
+Golang can exposes it's variables by expvar api, with this metricset, user can collect these metrics,
+By use parameter `expvar_keys` to specify which metrics to collect, by default or this parameter was not specified will
```

Comment:
@ruflin if the user didn't specify this config, the metrics will collect all the keys, are we still need to remove this config?